code/datums/components/twohanded.dm

```diff
@@ -160,8 +191,18 @@
 	// wield update status
 	if(SEND_SIGNAL(parent, COMSIG_TWOHANDED_WIELD, user) & COMPONENT_TWOHANDED_BLOCK_WIELD)
 		return // blocked wield from item
+
+	//If wielder isn't null already, unreference the old wielder
+	if(wielder != null)
+		unreference_wielder()
```

Comment:
This isn't needed, wielder will be null because of signal

Reply:
The item can be handed to another mob while still being wielded and without the wielder being deleted